dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -191,4 +191,11 @@
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="tool"/>
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="workflow"/>
     </changeSet>
+    <changeSet author="hyunnaye" id="fixConceptDOIs">
+        <sql dbms="postgresql">
+            UPDATE workflow
```

Comment:
Are you only updating the workflow table because it's the only one that has this issue? If somebody creates a DOI in app tool, service, or notebook between now and the 1.16 release, that won't introduce the same misformatted conceptdoi?

Reply:
oops, I added them for the other tables